lib/template.js

```diff
@@ -1,7 +1,7 @@
 const Handlebars = require('handlebars');
 const request = require('request');
 
-const ENV = require('../config/env');
+const ENV = require('../lib/env').env;
```

Comment:
`lib/env` is redundant with `env`, shall we call it `lib/config` ? (like on the fliplet-api)